api/src/main/java/com/alibaba/nacos/api/naming/pojo/Instance.java

```diff
@@ -211,6 +211,10 @@ private static boolean strEquals(final String str1, final String str2) {
         return str1 == null ? str2 == null : str1.equals(str2);
     }
     
+    public Long getInstanceHeartBeatIntervalNullable() {
```

Comment:
Is this too complicated？
I think one `containMetaData` is more simple, readable and versatile.